tests/src/FunctionalJavascript/SkipToMainContentTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_whitelabel\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+
+/**
+ * Verifies the "Skip to main content" link focuses <main id="main-content">.
+ */
+class SkipToMainContentTest extends WebDriverTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'oe_whitelabel';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'oe_bootstrap_theme_helper',
+  ];
+
+  /**
+   * Ensures the skip link focuses the main landmark.
+   */
+  public function testSkipToMainContentFocusesMain(): void {
+    $this->drupalGet('<front>');
+    $assert = $this->assertSession();
+    $skip_link = $assert->elementExists('css', 'a[href="#main-content"]');
+    $this->getSession()->executeScript(<<<'JS'
+      (function () {
+        var a = document.querySelector('a[href="#main-content"]');
+        if (!a) return;
+        a.focus();
+        a.click();
+      }());
+    JS);
```

Comment:
This does not capture how a user would find and interact with this link.
The goal is keyboard navigation:
- visit the page - at that point the skip link is hidden.
- hit "Tab" on the keyboard - at this point the skip link becomes visible, and has focus.
  (at some point we might want to assert the text of the link..)
- hit the enter key to trigger the link - this will focus `<main>`.

Reply:
this is what     $active_id = $this->getSession()->evaluateScript('return document.activeElement ? document.activeElement.id : "";'); does... gets the current focused element which should be #main-content. We assert in the test that focused element id is main-content    $this->assertSame('main-content', $active_id);... I dont see what we can do more than this.. it's the expected behaviour

Reply:
This is functional javascript test.. we test the behaviour of the skip link